src/reusedBrowser.ts

```diff
@@ -333,6 +333,14 @@ export class ReusedBrowser implements vscodeTypes.Disposable {
       );
       return false;
     }
+
+    if (this._vscode.env.uiKind === this._vscode.UIKind.Web) {
```

Comment:
Maybe instead when the user toggles the button? Or grey it out instead?

Reply:
Hiding it is a bit more involved.